app/src/main/java/com/example/mytrainingpal/screens/Home.kt

```diff
@@ -26,11 +26,18 @@ fun HomeScreen(navController: NavController) {
 
 @Composable
 fun HomeScreenContent(navigateToMusclePain: () -> Unit = {}, navigateToSettings: () -> Unit = {}) {
+
+    var wasPainEntered by remember { mutableStateOf(true) }
+    //TODO wasPainEntered should be updated in the model, toggled to true or false
+    // in that case we need a Model instance passed into HomeScreenContent
```

Comment:
For this I would suggest the following:
In the app's preferences (the app's personal key-value-store) I would save the date of the day the user entered sore muscles the last time. If the date is not today, we know that the prompt to enter the sore muscles should be shown.

Later (not part of this pr)
The person developing the sore muscle input needs to update this "last time entered sore muscles" date so the prompt disappears once the user enters their sore muscles.

Reply:
Should the app's preferences be declared and instantiated in the MainActivity file? 🤔

Reply:
Honestly: I have no idea. I haven't looked up, how the preferences work. I guess this link will be helpful: https://developer.android.com/develop/ui/views/components/settings/use-saved-values

Reply:
=> As it is better to use the room model classes to find out whether there is an entry for today, this pr is blocked by #21 and needs to wait until #21 is merged.